### PR TITLE
Add deployment configuration for digital-voucher suspension lambda

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This adds Cloudformation and Riffraff configuration to the project.

Riffraff has successfully [deployed](https://riffraff.gutools.co.uk/deployment/view/9c93355f-c2fe-4c3a-944c-68f66123d60e) the lambda to Code.

### CODE REVIEW

File: handlers/digital-voucher-suspension-processor/cfn.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    Description: Stage name
+    Type: String
+    AllowedValues:
+      - PROD
+      - CODE
+    Default: CODE
+
+Resources:
+
+  SuspensionLambda:
```
Comment (mario-galic) [inline handlers/digital-voucher-suspension-processor/cfn.yaml:15]: Where is description and role?

Comment (kelvin-chappell) [reply handlers/digital-voucher-suspension-processor/cfn.yaml:15]: This is a `Transform: AWS::Serverless-2016-10-31` template so the role is created implicitly.  
But I'll add a description.

**Review by mario-galic — APPROVED** [review]

File: handlers/digital-voucher-suspension-processor/cfn.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      Runtime: java8
+      MemorySize: 1536
+      Timeout: 900
+      CodeUri:
+        Bucket: support-service-lambdas-dist
+        Key: !Sub membership/${Stage}/digital-voucher-suspension-processor/digital-voucher-suspension-processor.jar
+      Handler: com.gu.digitalvouchersuspensionprocessor.Handler::handleRequest
+      Environment:
+        Variables:
+          salesforceUrl: !Sub '{{resolve:secretsmanager:digital-voucher-suspension-processor-${Stage}:SecretString:salesforceUrl}}'
```
Comment (ripecosta) [inline handlers/digital-voucher-suspension-processor/cfn.yaml:31]: Perhaps it's best to use versions here?

Comment (kelvin-chappell) [reply handlers/digital-voucher-suspension-processor/cfn.yaml:31]: After the recent discussion I think we can come up with a better way to manage versions.
I can't imagine any reason you would ever want to use anything but the latest version of a secret.

Comment (ripecosta) [reply handlers/digital-voucher-suspension-processor/cfn.yaml:31]: Agree that we probably always want the latest version of a secret. And it's very likely we'll come up with that better way to manage version before we need to change this lambda's credentials. So yeah 👍

**Pull request merged**